libs/jsonschema/schema.go

```diff
@@ -153,6 +169,52 @@ func (schema *Schema) validate() error {
 	return nil
 }
 
+func (schema *Schema) validateSchemaMinimumCliVersion(currentVersion string) func() error {
+	return func() error {
+		if schema.MinDatabricksCliVersion == "" {
+			return nil
+		}
+
+		// Ignore this validation rule for local builds.
+		if semver.Compare("v"+build.DefaultSemver, currentVersion) == 0 {
+			return nil
+		}
+
+		// Confirm that MinDatabricksCliVersion starts with a 'v'.
+		if !strings.HasPrefix(schema.MinDatabricksCliVersion, "v") {
```

Comment:
Why do we need this if we call `semver.IsValid` below?

Reply:
@shreyas-goenka we could combine these 2 checks and have better errors message like `invalid minimum CLI version %q specified. Please specify the version in the format v0.0.0`

Otherwise version might start with v but still be invalid like, `v0.0.0.` or smth

Reply:
Good point! It's much clearer this way.